src/entities/inventory.entity.js

```diff
@@ -0,0 +1,13 @@
+class Inventory {
```

Comment:
Don't you need to export the class to make it visible to other layers of the project?